### PR TITLE
command for covering head switching noise resolving #42

### PR DESCRIPTION
**Pull request by ablwr**
Hello! This one had been lingering around for quite a while! Free time is great to have!

### CODE REVIEW

**Comment by kieranjol** [comment]
Looks great to me, and really useful!

**Comment by dericed** [comment]
But why cover it? Why not just crop it off. btw this command, shows the bottom 20 lines with magnification and a grid overlay so it's easier to count how many lines to conceal

```
ffplay some_analogy_vid.mkv -vf crop=iw:20:0:ih-20,scale=iw:ih*20:flags=neighbor,setdar=4/3,drawgrid=width=iw:height=ih/20:t=1:c=yellow
```

**Comment by ablwr** [comment]
If this is somewhat a vanity issue and is for access/playback, wouldn't the crop-off potentially result in finicky video playback -- potentially causing stretch/distortion?

I like this ffplay command though, maybe can integrate into the existing or merits its own script (it's kind of complex to explain out).

**Comment by dericed** [comment]
With cropping the SAR will remain the same, though the DAR will change accordingly, so it shouldn't be stretch by any player that can understand the sar.

File: index.html [hunk 1]
```diff
@@ -537,6 +537,36 @@
...
   <!-- ends Create ISO -->
 
+  <!-- Cover head switching noise -->
+  <span data-toggle="modal" data-target="#cover_head"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Cover head switching noise">Cover head</button></span>
+  <div id="cover_head" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Cover head switching noise</h3>
+          <p><code>ffmpeg -i <i>input_file</i> -vf drawbox=w=iw:h=7:y=ih-h:t=max <i>output_file</i></code></p>
```
Comment (retokromer) [inline index.html]: In my opinion, it should be said that `-vf` in an alias of `-filter:v`.

File: index.html [hunk 2]
```diff
@@ -537,6 +537,36 @@
...
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-i <i>input_file</i></dt><dd>path, name and extension of the input file</dd>
+            <dt>-vf drawbox=</dt>
+            <dd>This calls the drawtext filter with the following options:
+              <dl>
+                <dt>w=in_w</dt><dd>Width is set to the input width. Shorthand for this command would be w=iw</dd>
+                <dt>h=7</dt><dd>Height is set to 7 pixels.</dd>
+                <dt>y=ih-h</dt><dd>Y represents the offset, and ih-h sets it to the input height minus the height declared in the previous parameter, setting the box at the bottom of the frame.</dd>
+                <dt>t=max</dt><dd>T represents the thickness of the drawn box. Default is 3. </dd>
```
Comment (retokromer) [inline index.html]: Space before `</dd>` can be deleted.

File: index.html [hunk 3]
```diff
@@ -537,6 +537,36 @@
...
+              </dl>
+            </dd>
+            <dt><i>output_file</i></dt><dd>path and name of the output file.</dd>
```
Comment (retokromer) [inline index.html]: In the other recipes we closed only full sentence with a period. I suggest to cancel it here.

**Comment by ablwr** [comment]
Thanks always, R E T O !

**Comment by retokromer** [comment]
I merge. If the discussion comes to another result we can revert.

**Comment by ablwr** [comment]
Maybe @dericed can add his above script. 👼

**Pull request merged**